main.py

```diff
@@ -0,0 +1,58 @@
+from fastapi import FastAPI, HTTPException, status
+from schemas.post import Post, ResponsePost, RequestPost
+
+app = FastAPI()
+post_data = []
+
+
+@app.get("/", status_code=status.HTTP_200_OK)
+def read_root() -> dict:
```

Comment:
```suggestion
def read_root() -> dict[str, str]:
```

위처럼 dict, list 등 element에도 type을 추가하곤 합니다 ㅎㅎ

Reply:
감사합니다 적용해보겠습니다 :)